main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -127,6 +127,8 @@ object DocAst {
 
     case class Native(clazz: Class[_]) extends Atom
 
+    case class StructNew(sym: Symbol.StructSym, exps: List[(Name.Label, Expr)], exp: Expr) extends Composite
```

Comment:
Sorry for the slow review

This should be expressible with `DoubleKeyword("new", exps.foldRight(RecordEmpty: Expr){ case (acc, cur) => RecordExtend(cur._1, cur._2, acc)}, "@", d2)`

Reply:
or something close to that

Reply:
I would need `new StructName {...} @ r`. Can I just concatenate the struct name after the "new" in `DoubleKeyword`?

Reply:
oh yeah - yes that works